diffs/filebeat2.diff

```diff
@@ -0,0 +1,4093 @@
+                  proxy_url
+@@ -1299,7 +1299,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_max_retries_literal">
++                <a id="_max_retries">
+                 </a>
+                 <code class="literal">
+                  max_retries
+@@ -1324,7 +1324,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_bulk_max_size_literal">
++                <a id="_bulk_max_size">
+                 </a>
+                 <code class="literal">
+                  bulk_max_size
+@@ -1362,7 +1362,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_backoff_init_literal">
++                <a id="_backoff_init">
+                 </a>
+                 <code class="literal">
+                  backoff.init
+@@ -1391,7 +1391,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_backoff_max_literal">
++                <a id="_backoff_max">
+                 </a>
+                 <code class="literal">
+                  backoff.max
+@@ -1412,7 +1412,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_timeout_literal">
++                <a id="_timeout">
+                 </a>
+                 <code class="literal">
+                  timeout
+@@ -1433,7 +1433,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_ssl_literal">
++                <a id="_ssl">
+                 </a>
+                 <code class="literal">
+                  ssl
+html/en/beats/filebeat/master/enable-filebeat-debugging.html
+html/en/beats/filebeat/master/enroll-beats.html
+html/en/beats/filebeat/master/error-found-unexpected-character.html
+html/en/beats/filebeat/master/error-loading-config.html
+html/en/beats/filebeat/master/exported-fields-apache.html
+html/en/beats/filebeat/master/exported-fields-auditd.html
+html/en/beats/filebeat/master/exported-fields-aws.html
+html/en/beats/filebeat/master/exported-fields-beat-common.html
+html/en/beats/filebeat/master/exported-fields-cef-module.html
+html/en/beats/filebeat/master/exported-fields-cef.html
+html/en/beats/filebeat/master/exported-fields-cisco.html
+html/en/beats/filebeat/master/exported-fields-cloud.html
+html/en/beats/filebeat/master/exported-fields-coredns.html
+html/en/beats/filebeat/master/exported-fields-docker-processor.html
+html/en/beats/filebeat/master/exported-fields-ecs.html
+html/en/beats/filebeat/master/exported-fields-elasticsearch.html
+--- /dev/fd/63
++++ html/en/beats/filebeat/master/exported-fields-elasticsearch.html
+@@ -538,7 +538,7 @@
+               REST URI parameters
+              </p>
+              <p class="simpara">
+-              example: jacknich2
++              example: {username=jacknich2}

```

Comment:
possible fix required

Reply:
this is intended behavior; the previous behavior of asciidoc resolving the example as an attribute was not expected